### DISCUSSION TITLE
[Support]: bug? in the new ui

### DISCUSSION THREAD

**Discussion by JustinScott** [blakeblackshear/frigate#11654] [General Support] [answered]
### Describe the problem you are having

On the main page with the live camera's, occasionally, the images for the cameras are very small and lined up vertically. It seems like it's the mobile version of the webpage. If I refresh a few times the page is displayed normally.

### Version

dev-4135cab

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### FFprobe output from your camera

```shell
n/a
```

### Operating system

UNRAID

### Install method

HassOS Addon

### Coral version

USB

### Network connection

Wired

### Camera make and model

n/a

### Any other information that may be helpful

Firefox 127.0b8 (64-bit)

![Screenshot 2024-05-30 112838](https://github.com/blakeblackshear/frigate/assets/1866338/1db4f87d-ad1c-4a21-b656-0b825efcc499)

**Comment by NickM-27** [comment]
you may want to try refreshing the layout in settings

**Comment by JustinScott** [comment]
You mean "Clear All Layout" under stored layouts? I clicked it several times and the problem is still there.

**Comment by JustinScott** [comment]
It seems to only happen on a camera group I created. Not the default group. If I f5 a few times on that group, the problem shows up and a few f5s later disappears.

**Reply by hawkeye217** [reply]
I've only ever seen this when I update and restart Frigate. If I switch to the default group and switch back to my camera group, everything looks fine. Did you recently update or restart?

**Reply by JustinScott** [reply]
I don't think uptime matters. My Frigate uptime is 1 hour right now and I can reproduce the bug. I'll try it again later today.

@hawkeye217 Are you able to reproduce the bug by spamming f5 a few times on the custom group page?

**Reply by hawkeye217** [reply, marked as answer]
It's a race condition, and I can reproduce it too. Thanks for the report.

**Comment by hawkeye217** [comment]
This is fixed in https://github.com/blakeblackshear/frigate/pull/11656

**Comment by JustinScott** [comment]
Just experienced this bug again. Happened right after a restart.

build: dev-402c16e

**Reply by hawkeye217** [reply]
After a restart makes sense, and because of the way your browser stores the camera groups and layouts, a reload of the UI is always required. If you still experience it otherwise, let me know.
